nautobot/core/sso.py

```diff
@@ -0,0 +1,75 @@
+"""Customized Nautobot SSO items."""
+from constance import config
+from django.conf import settings
+from django.shortcuts import resolve_url
+from django.utils.encoding import force_str
+from django.utils.functional import Promise
+from social_core.pipeline import DEFAULT_AUTH_PIPELINE
+from social_django.strategy import DjangoStrategy
+
+from nautobot.users.models import SSOBackend
+
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class NautobotStrategy(DjangoStrategy):
+    """Subclassing to allow for constance config exposed via Nautobot."""
```

Comment:
Can you please revise this docstring to describe how it is used? It's not clear act a `Strategy` is.